docs/reference/search/multi-search.asciidoc

```diff
@@ -86,11 +86,10 @@ the maximum number of concurrent searches the multi search api will execute.
 This default is based on the number of data nodes and the default search thread pool size.
 
 The request parameter `max_concurrent_shard_requests` can be used to control the
-maximum number of concurrent shard requests the each sub search request will execute.
+maximum number of concurrent shard requests the each sub search request will execute per node.
 This parameter should be used to protect a single request from overloading a cluster
 (e.g., a default request will hit all indices in a cluster which could cause shard request rejections
-if the number of shards per node is high). This default is based on the number of
-data nodes in the cluster but at most `256`.In certain scenarios parallelism isn't achieved through
+if the number of shards per node is high). This default value is `5`.In certain scenarios searcparallelism isn't achieved through
```

Comment:
nit: s/In certain scenarios searcparallelism/In certain scenarios parallelism/

Reply:
bla :) will fix